index.js

```diff
@@ -0,0 +1,105 @@
+// eslint-disable-next-line no-unused-vars
+import date from './modules/dateAndTime.js';
+import Book from './modules/classBook.js';
+
+const addTitle = document.querySelector('#addTitle');
+const addAuthor = document.querySelector('#addAuthor');
+const addForm = document.querySelector('#addBook');
+const allBooks = document.querySelector('.books');
+const listMenu = document.querySelector('#list-section');
+const addBookMenu = document.querySelector('#add-book-section');
+const contactMenu = document.querySelector('#contact-section');
+const listSection = document.querySelector('.main');
+const addBookSection = document.querySelector('.formSection');
+const contactSection = document.querySelector('.contact');
+
+let bookCollection = [];
+
+let newBook;
+let formData;
+let bookCollectionHtml;
+class AddRemove {
+  static addBookToStorage() {
+    const str = JSON.stringify(bookCollection);
+    localStorage.setItem('storedBookData', str);
+  }
+
+  static deleteBook(id) {
+    const itemToDelete = bookCollection[id];
+
+    bookCollection = bookCollection.filter((item) => item !== itemToDelete);
+  }
+}
+
+const addBtnRemoveEvent = () => {
+  document.querySelectorAll('.delete_btn').forEach((button) => button.addEventListener('click', (event) => {
+    event.preventDefault();
+    const { id } = button;
+    AddRemove.deleteBook(id);
+    AddRemove.addBookToStorage();
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Kindly remove this line and fix the linters because it makes you follow best practices and your code clean.